tests/internal/e2e/test/specs/vcast_coded_test_completion.test.ts

```diff
@@ -0,0 +1,416 @@
+
+    await $(
+      ".setting-item-description*=Decorate files that have coverage in the File Explorer pane"
+    );
+    await editorView.closeEditor("Settings");
+  });
+
+  it("should check vor vmock code completion", async () => {
+    await updateTestID();
+
+    console.log("Opening Testing View");
+    const vcastTestingViewContent = await getViewContent("Testing");
+    let subprogram: TreeItem;
+
+    // Expand manager section in testing pane.
+    for (const vcastTestingViewSection of await vcastTestingViewContent.getSections()) {
+      if (!(await vcastTestingViewSection.isExpanded()))
+        await vcastTestingViewSection.expand();
+
+      for (const vcastTestingViewContentSection of await vcastTestingViewContent.getSections()) {
+        console.log(await vcastTestingViewContentSection.getTitle());
+        await vcastTestingViewContentSection.expand();
+        subprogram = await findSubprogram(
+          "manager",
+          vcastTestingViewContentSection
+        );
+        if (subprogram) {
+          if (!(await subprogram.isExpanded())) await subprogram.expand();
+          break;
+        }
+      }
+    }
+
+    if (!subprogram) {
+      throw "Subprogram 'manager' not found";
+    }
+
+    console.log("Looking for coded tests");
+    const subprogramMethod = await findSubprogramMethod(
+      subprogram,
+      "Coded Tests"
+    );
+    if (!subprogramMethod) {
+      throw "Subprogram method 'Coded Tests' not found";
+    }
+
+    if (!subprogramMethod.isExpanded()) {
+      await subprogramMethod.select();
+    }
+
+    let contextMenu = await subprogramMethod.openContextMenu();
+    console.log("Generating template test");
+    await contextMenu.select("VectorCAST");
+    let menuElement = await $("aria/Generate New Coded Test File");
+    await menuElement.click();
+
+    await (await $("aria/Save Code Test File")).click();
+    for (const character of "TestFiles/manager-template.cpp") {
+      await browser.keys(character);
+    }
+
+    await browser.keys(Key.Enter);
+
+    await bottomBar.openOutputView();
+    console.log("Checking that tests got generated");
+    let testHandle = await getTestHandle(
+      subprogram,
+      "Coded Tests",
+      "managerTests.ExampleFixtureTestCase",
+      2
+    );
+    expect(testHandle).not.toBe(undefined);
+
+    testHandle = await getTestHandle(
+      subprogram,
+      "Coded Tests",
+      "managerTests.ExampleTestCase",
+      2
+    );
+    expect(testHandle).not.toBe(undefined);
+
+    contextMenu = await testHandle.openContextMenu();
+    await contextMenu.select("VectorCAST");
+    menuElement = await $("aria/Edit Coded Test");
+    await menuElement.click();
+
+    const editorView = workbench.getEditorView();
+    await browser.waitUntil(
+      async () =>
+        (await (await editorView.getActiveTab()).getTitle()) ===
+        "manager-template.cpp"
+    );
+
+    // Insert "// vmock" on line 16
```

Comment:
adapt comment to match the code (line 14)

Reply:
Refactored in b797c7864c92738222537ffd5e50d5fc216835ba .